logging/ContextLogPropertiesWrapper.ts

```diff
@@ -0,0 +1,18 @@
+import {LogPropertiesWrapper, PolarisLogProperties} from "@enigmatis/polaris-logs";
+import {GraphQLLogProperties} from "./GraphQLLogProperties";
+
+export class ContextLogPropertiesWrapper implements LogPropertiesWrapper{
```

Comment:
delete this class, since LogPropertiesWrapper will be deleted soon

Reply:
done